lib/config_files/base_model.rb

```diff
@@ -38,5 +69,77 @@ def merge_changes
       # TODO: recursive merge
       data.merge(new_data)
     end
+
+    def modify(key, value)
+      # if already set, just change value
+      return false unless data[key]
+
+      data[key] = value
+      true
+    end
+
+    def uncomment(key, value)
+      # Try to find if it is commented out, so we can replace line
+      matcher = Matcher.new(
+        collection:    "#comment",
+        value_matcher: /#{key}\s*=/
```

Comment:
you are right, also I am not sure if all lenses uses "#comment" so my plan is to override it if needed. Regarding similar names, you are right, maybe use something like `/(\s|^)#{key}.../`?